server/http.go

```diff
@@ -87,17 +100,21 @@ func (p *Plugin) handleStandardWebhook(w http.ResponseWriter, r *http.Request, w
 	post.Message = "Meeting has ended."
 	post.Props["meeting_status"] = zoom.WebhookStatusEnded
 
-	if _, appErr := p.API.UpdatePost(post); appErr != nil {
+	_, appErr = p.API.UpdatePost(post)
+	if appErr != nil {
```

Comment:
I'm not sure if this makes the code more readable. What is your reasoning behind the change?

Reply:
It is personally easier for me to scan/read, but I understand if it is not for others. I assume it is common in the Go community to use the `if ...;` syntax. Though using separate lines is a convention the integrations team has made for any new code we introduce across the plugins we maintain.

Reply:
How strongly do you feel about changing the blocks back to the `if ...;` syntax?

Reply:
Good question. 0/5, it's personal preference.